client/src/components/home/HomePage.tsx

```diff
@@ -159,19 +165,28 @@ export default class HomePage extends Component<IHomePageProps, IHomePageState>
         <Row />
         <Row>
           <Col md={12}>
-            <FormGroup bsSize="large" controlId="address" validationState={this.getValidationState()}>
+            <FormGroup bsSize="large" controlId="address" validationState={this.getValidationState().validation}>
```

Comment:
If you make #getValidationState return simply Validation type, you can write
`validationState={this.getValidationState()}`